databasetest.py

```diff
@@ -0,0 +1,21 @@
+import getpass
+import mysql.connector
+
+
+if __name__ == '__main__':
+    db_password = getpass.getpass()
+    #change the user to whatever ur calpoly email is, leave database the same
+    conn = mysql.connector.connect(user='akpravee', password=db_password,
```

Comment:
Switch user, host, and database to variables you set above and refer to in case we ever need that again